File: Project-Euler/Question-1/Multiples_of_3_and_5.py
# number_1 = 3
# number_2 = 5 
current_number = 1
total = 0
upper_number = 1000

def meat(current_number,total,upper_number):
    while current_number < upper_number:
        if current_number % 3 == 0 or current_number % 5 == 0:
            total = total + current_number
            
        current_number = current_number +  1

    return total

#main

total = meat(current_number,total,upper_number)
print(total)

File: Project-Euler/Question-1/Multiples_of_3_and_5.rs
fn main()
{
    let mut current_number = 1;
    let mut total = 0;
    let upper_number = 1000;
    
    while current_number < upper_number
    {
        if current_number %3 == 0 || current_number % 5 == 0
        {
            total = total + current_number;
        } 
        
        current_number = current_number + 1
    }
    
    println!("{}",total);
}
